backend/src/main.py

```
@@ -81,12 +72,119 @@ def internal_error(error):
+    """
+    Starts an asyncio event loop in a background thread for monitoring and auto-trading.
+    """
+    global background_loop
+    background_loop = asyncio.new_event_loop()
+    asyncio.set_event_loop(background_loop)
+
+    # Inform services about the background loop
+    auto_trader_service.set_loop(background_loop)
+
+    # Run post_init in the background loop
+    background_loop.create_task(auto_trader_service.post_init())
```

Ruff (0.15.13)

[warning] Store a reference to the return value of `background_loop.create_task`

(RUF006)